### ISSUE TITLE
Accuracy will change the model's output during validation

### ISSUE THREAD

**Issue by qiuxin2012** [intel/analytics-zoo#780]
`com.intel.analytics.zoo.pipeline.api.keras.metrics.Accuracy` will change the model's output(in BigDL's Top1Accuracy), if we use a AUC after Top1Accuracy will lead to a wrong AUC score.
